File: src/front/client/src/app/Models/Weather.ts
export class Weather {
  public name: string;
    public description: string;
    public icon: string;
    public temperature: number;
    public minTemperature: number;
    public maxTemperature: number;
    public humidity: number;
    public wind: number;

  constructor(
    name: string,
     description: string,
     icon: string,
     temperature: number,
     minTemperature: number,
     maxTemperature: number,
     humidity: number,
     wind: number
  ) {
    this.name = name;
    this.description = description;
    this.icon = icon;
    this.temperature = temperature;
    this.minTemperature = minTemperature;
    this.maxTemperature = maxTemperature;
    this.humidity = humidity;
    this.wind = wind;
  }
}

File: src/back/API/Services/WeatherForecast/Models/Weather.cs
namespace API.Services.WeatherForecast.Models;

public class Weather
{
    public string? Description { get; set; }
    public string? IconUrl { get; set; }
    public double? Temperature { get; set; }
    public double? MinTemperature { get; set; }
    public double? MaxTemperature { get; set; }
    public double? Humidity { get; set; }
    public double? WindSpeed { get; set; }
}